### PR TITLE
fix: updated android and ios sdk versions

### PR DESCRIPTION
**Pull request by Subhash703**

### CODE REVIEW

File: example/android/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "33.0.0"
-        minSdkVersion = 21
+        minSdkVersion = 22
```
Comment (harshgarg18) [inline example/android/build.gradle:6]: why is this updated?

File: example/android/build.gradle [hunk 2]
```diff
@@ -12,7 +12,8 @@ buildscript {
 
         // Hyper SDK
         clientId = 'geddit'
-        hyperSDKVersion = "2.1.15"
+        hyperSDKVersion = "2.1.30"
+        excludedMicroSDKs = []
```
Comment (harshgarg18) [inline example/android/build.gradle:16]: This is a very niche use case. We should not put this by default in our demo app

File: example/android/app/build.gradle [hunk 3]
```diff
@@ -146,6 +146,13 @@ android {
 
         }
     }
+    configurations.configureEach {
```
Comment (harshgarg18) [inline example/android/app/build.gradle:149]: note to self: needs to be evaluated

File: example/android/build.gradle (hunk 1 above)
Comment (Subhash703) [reply example/android/build.gradle:6]: default clientId is `geddit`, which have hyperapay module (not supported on 21)

**Comment by harshgarg18** [comment]
Stale PR

**Pull request closed without merging**